cdc-accelerator/feature/acls.js

```diff
@@ -0,0 +1,70 @@
+import PartnerFeature from './partnerFeature.js'
+import path from 'path'
+import fs from 'fs'
+import { clearDirectoryContents } from '../utils/utils.js'
+import { SRC_DIRECTORY, BUILD_DIRECTORY } from './constants.js'
+import client from '../../cdc-accelerator/sap-cdc-toolkit/gigya/client.js'
+export default class ACL extends PartnerFeature {
```

Comment:
ACL class needs to extend PartnerFeature? 
It is a feature on it's own or is dependent of the PermissionGroups?